site/docs/multi-engine-support.md

```diff
@@ -42,6 +42,10 @@ Iceberg provides a runtime connector jar for each supported version of Spark, Fl
 When using Iceberg with these engines, the runtime jar is the only addition to the classpath needed in addition to vendor dependencies.
 For example, to use Iceberg with Spark 3.5 and AWS integrations, `iceberg-spark-runtime-3.5_2.12` and AWS SDK dependencies are needed for the Spark installation.
 
+> ℹ️ It's important to make sure that only the runtime jars (plus storage specific bundles if needed, eg. `iceberg-aws-bundle` or `iceberg-gcp-bundle`) are included in the runtime classpath.
```

Comment:
how does this page render? Can you provide a screenshot when running the site locally?